setup.py

```diff
@@ -13,6 +13,18 @@
     use_setuptools()
     from setuptools import setup, find_packages
 
+install_requires = [
+    'wagtail>=2.9',
```

Comment:
As the maintainer probably won't be considering this work we should use.

```suggestion
    'wagtail>=4.1,
```

Reply:
Fixed here: https://github.com/torchbox-forks/wagtail-accessibility/pull/1/commits/d153b1d99478b27b1d305935c4410d8a92437842

Reply:
Excellent 👍